x/upgrade/module.go

```diff
@@ -59,10 +58,7 @@ func (AppModuleBasic) GetQueryCmd(clientCtx client.Context) *cobra.Command {
 		Use:   "upgrade",
```

Comment:
I´d move this to "github.com/cosmos/cosmos-sdk/x/upgrade/client/cli"

Reply:
Done, did the same thing for GetTxCmd.